packages/web/src/upstream/user-interaction/privacy/privacy-manager.ts

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
would it be possible to put this in managers folder? this is not really an upstream thing